/**:
  ros__parameters:
    repeat: 1
    max_velocity: 0.5
    max_acceleration: 0.2
    velocity_scaling_list: [0.3, 0.5, 0.7]
    accelration_scaling_list: [0.1, 0.3, 0.5]